tests/role_wrapper.yml

```diff
@@ -0,0 +1,105 @@
+- name: Check that tests_role_name variable is set
+  assert:
+    that:
+      - tests_role_name is defined
+      - tests_role_name | length > 0
+    fail_msg: >
+      tests_role_name variable must be set and not empty.
+  changed_when: false
+  delegate_to: localhost
+
+- name: >
+    Check that tests_test_checkmode and tests_test_idempotency are not both set
+  assert:
+    that: not (
+        ( tests_test_checkmode | default(false) ) and
+        ( tests_test_idempotency | default(false) )
+      )
+    fail_msg: >
+      Either tests_test_checkmode or tests_test_idempotency can be enabled, but
+      not both.
+  changed_when: false
+  delegate_to: localhost
+
+- name: Create a temporary file for a static inventory
+  tempfile:
+    state: file
+    suffix: .inventory.yml
+  register: tempinventory
+  changed_when: false
+  delegate_to: localhost
+
+- name: Create a static inventory from hostvars
+  template:
+    src: inventory.yml.j2
+    dest: "{{ tempinventory.path }}"
+  changed_when: false
+  delegate_to: localhost
+  when:
+    - tempinventory is defined
+    - tempinventory.path is defined
+
+- block:
+    - name: Import role {{ tests_role_name }}
+      import_role:
+        name: "{{ tests_role_name }}"
```

Comment:
What is this import_role for?  Is this just to verify that the given role can be imported?